microWorker.js

```diff
@@ -190,6 +193,80 @@ function _setupFiles(bag, next) {
   );
 }
 
+function _setExecutorAsReqKick(bag, next) {
+  var who = bag.who + '|' + _setExecutorAsReqKick.name;
+  logger.verbose(who, 'Inside');
+
+  var whoPath = util.format('%s/job.who', bag.buildStatusDir);
+  fs.writeFile(whoPath, 'reqKick\n',
+    function (err) {
+      if (err) {
+        bag.jobStatusCode = __getStatusCodeByName('error', bag.isCI);
+        return next();
+      }
+
+      return next();
+    }
+  );
+}
+
+function _pollExecutorForReqProc(bag, next) {
+  var who = bag.who + '|' + _pollExecutorForReqProc.name;
+  logger.verbose(who, 'Inside');
+
+  function checkForReqProc(bag, callback) {
+    var whoPath = util.format('%s/job.who', bag.buildStatusDir);
+    var isReqProc = false;
+
+    try {
+      var executor = fs.readFileSync(whoPath, {encoding: 'utf8'});
+      isReqProc = executor.trim() === 'reqProc';
+    } catch (err) {
+      isReqProc = false;
+    }
+
+    if (isReqProc)
+      return callback();
+
+    setTimeout(function () {
```

Comment:
Will this be cleared automatically?

Reply:
yes! it will clear. We are using similar kind of logic in lots of places in deploy job 
Example: https://github.com/Shippable/micro/blob/master/nod/genExec/managed/deploy/providers/ecs/common/blueGreen.js#L908-L913

Reply:
Okay, works. Let's update this to chokidar later.